pegasus/cache/i18n/en-US.yml

```diff
@@ -1520,6 +1520,7 @@
   full_course_category_name: "Full Courses"
   user_header_label: "Hi %{username}"
   user_header_team: "Team"
+  user_header_my_projects: "My projects"
```

Comment:
Yes, this will be overwritten when we do a content scoop on the staging machine as we pull changes made to the gsheet.  Instead, this should be modified in the gsheet  first,and we should ensure that the staging branch contains the scooped change before this PR is merged.

Reply:
OK, I've added it to the spreadsheet, line 1492 (http://sheets/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU#gid=0)

Reply:
Great.  You can confirm everything has worked by waiting for that change to be scooped, and then merge the staging branch into this branch, and the above line should vanish.  (If it's still there, we'll have to remove it so we don't have a double-entry.)

Reply:
I've merged with head, and it looks to me like this change has gone away, so we should be good to go.